evm/utils.js

```diff
@@ -1083,12 +1084,24 @@ async function printTokenInfo(tokenAddress, provider) {
 
 async function isTrustedChain(destinationChain, interchainTokenService, itsVersion) {
     if (itsVersion === '2.1.1') {
-        return ((await interchainTokenService.trustedAddress(destinationChain)) !== '');
+        return (await interchainTokenService.trustedAddress(destinationChain)) !== '';
     } else {
-        return ((await interchainTokenService.isTrustedChain(destinationChain)));
+        return await interchainTokenService.isTrustedChain(destinationChain);
     }
 }
 
+function detectITSVersion(options) {
+    let itsVersion;
+
+    if (options.itsVersion) {
```

Comment:
it's always going to be the dependency version that is being deployed. so, allowing an override seems confusing and potentially error-prone. Let's avoid adding code we don't need yet